Buildings/Examples/VAVReheat/Controls/ModeSelector.mo

```diff
@@ -95,19 +94,34 @@ model ModeSelector "Finite State Machine for the operational modes"
   Modelica.StateGraph.TransitionWithSignal t8
     "changes to occupied in case precooling is deactivated"
     annotation (Placement(transformation(extent={{30,-30},{50,-10}})));
-  Modelica.Blocks.MathInteger.Sum sum(nu=5)
-    annotation (Placement(transformation(extent={{-186,134},{-174,146}})));
+  Modelica.Blocks.MathInteger.Sum sum(nu=6)
+    annotation (Placement(transformation(extent={{-192,134},{-180,146}})));
   Modelica.Blocks.Interfaces.BooleanOutput yFan
     "True if the fans are to be switched on"
-    annotation (Placement(transformation(extent={{220,-10},{240,10}})));
+    annotation (Placement(transformation(extent={{220,80},{260,120}}),
+        iconTransformation(extent={{220,80},{260,120}})));
   Modelica.Blocks.MathBoolean.Or or1(nu=4)
-    annotation (Placement(transformation(extent={{184,-6},{196,6}})));
+    annotation (Placement(transformation(extent={{160,-10},{180,10}})));
+  Modelica.Blocks.Interfaces.BooleanOutput yEco
+    "True if the economizer is enabled" annotation (Placement(transformation(
```

Comment:
Maybe better named `yEcoSta`?

Reply:
I would prefer not to use status for a command signal (digital output) and reserve its use for the return signal from the equipment (digital input). This is consistent with Guideline 36 and RP1711 I believe.